src/components/StatusBar/index.js

```diff
@@ -14,7 +15,7 @@ const styles = StyleSheet.create({
     height: STATUSBAR_HEIGHT,
   },
   appBar: {
-    backgroundColor: '#79B45D',
+    backgroundColor: COLORS.greenMantis,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from hard-coded color value to using a constant from the `COLORS` object is a good practice for maintainability and consistency. However, it's important to ensure that `COLORS.greenMantis` is defined and has the same value as `'#79B45D'`. If not, this could lead to unexpected UI changes.
```diff
-    backgroundColor: '#79B45D',
+    backgroundColor: COLORS.greenMantis,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->